bundle/hive-package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+- currentCSV: hive-operator.v1.0.0-1
+  name: alpha
+defaultChannel: alpha
+packageName: hive-operator
```

Comment:
So our bundle/ dir will just grow over time with each new version, and this file is the central entrypoint for the version graph, do I have that right? This is quite cool if so.

Reply:
yes